apps/global_preferences.cpp

```diff
@@ -44,3 +46,13 @@ void GlobalPreferences::setBrightnessLevel(int brightnessLevel) {
     Ion::Backlight::setBrightness(m_brightnessLevel);
   }
 }
+
+void GlobalPreferences::setPythonHeap(uint8_t n)
+{
+  this->python_heap = ((n * 46) / 21) + 16;
+}
+
+int8_t GlobalPreferences::getPythonHeap()
+{
+  return ((this->python_heap - 16) * 21) / 46;
+}
```

Comment:
Can you add line return at the end of all files ?
```suggestion
}

```